src/com/rockbite/bootcamp/store/IInventory.java

```diff
@@ -0,0 +1,12 @@
+package com.rockbite.bootcamp.store;
+
+import com.rockbite.bootcamp.store.collections.Resources.Item;
+
+public interface IInventory {
+
+    boolean hasItem(Item item, int count);
+
+    void spendItem(Item item, int count);
```

Comment:
spendItem, suggests that you are paying for something with an item. What are you purchasing in exchange for this?